### DISCUSSION TITLE
deploying on netlify

### DISCUSSION THREAD

**Discussion by usvyaty** [lowdefy/lowdefy#1036] [Questions and Help] [answered]
When I deploy on Netlify I get :
7:19:22 PM:   Configuration error                                           
7:19:22 PM: ────────────────────────────────────────────────────────────────
7:19:22 PM: ​
7:19:22 PM:   Error message
7:19:22 PM:   **Deploy did not succeed: Deploy directory '.lowdefy/publish' does not exist**
7:19:22 PM: ​
7:19:22 PM:   Resolved config
7:19:22 PM:   build:
7:19:22 PM:     command: npx lowdefy@latest build-netlify
7:19:22 PM:     commandOrigin: ui
7:19:22 PM:     publish: /opt/build/repo/.lowdefy/publish
7:19:22 PM:     publishOrigin: ui
7:19:22 PM:   functionsDirectory: /opt/build/repo/.lowdefy/function

Any idea why?

Thank you.

**Comment by SamTolmay** [comment, marked as answer]
Hi @usvyaty 

It looks like the Lowdefy build failed, or did not build. Are there any error messages or warnings a little earlier in the logs?

**Comment by usvyaty** [comment]
Hi @SamTolmay ,
Thank you for getting back. there was indeed a build error which I missed.

Thanks a million.
